toolbar_utils.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2011, Walter Bender
-# Port To GTK3:
-# Ignacio Rodriguez <ignaciorodriguez@sugarlabs.org>
+# Copyright (C) 2011, Walter Bender
+# Copyright (C) Ignacio Rodriguez <ignaciorodriguez@sugarlabs.org>
+#
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
 # You should have received a copy of the GNU General Public License
-# along with this library; if not, write to the Free Software
-# Foundation, 51 Franklin Street, Suite 500 Boston, MA 02110-1335 USA
-
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
+import gi
+gi.require_version('Gtk', '3.0')
```

Comment:
import and require_version not needed here, because it is already done for the process in CardSortActivity.py